mlflow/recipes/steps/train.py

```diff
@@ -1023,7 +1023,7 @@ def render_schema(inputs, title):
         model_url = get_databricks_run_url(
             tracking_uri=mlflow.get_tracking_uri(),
             run_id=run_id,
-            artifact_path=re.sub(r"^.*?%s" % run_id, "", model_uri_path),
+            artifact_path=re.sub(r"^.*?{run_id}", "", model_uri_path),
```

Comment:
Starnge. Did the script make this change?

Reply:
Did you manually remove `f` or not?

Reply:
Thanks for taking a look at this. This has been updated to keep the r-formatted string. Using the raw string will keep us from having to escape `model_uri_path`, will have to pass on the f-string here